### ISSUE TITLE
Add support for migrations

### ISSUE THREAD

**Issue by swaffoja** [mogol/flutter_realm#6]
Do you have any thoughts on implementation for [migrations](https://realm.io/docs/objc/latest/#migrations)? Passing schema version through the [configuration](https://github.com/mogol/flutter_realm/blob/master/lib/src/realm.dart#L224) is straightforward. The migration block would need to exist in native Java and ObjC host apps much like the model...

**Comment by boring2** [comment]
@swaffoja  So the library not support realm migrations??

**Comment by swaffoja** [comment]
@boring2 I do not think migrations are currently supported.

**Comment by ryanbliss** [comment]
Yeah this would be a really big one for most projects. Months into development this could bite you, so I definitely second this request.

**Comment by gonharry** [comment]
+1, if this plugin can do that it would be great.

**Comment by so0602** [comment]
+2, if this plugin can do that it would be great.

**Comment by swaffoja** [comment]
I have abandoned hope and development effort for Realm in Flutter. Check out [Moor](https://moor.simonbinder.eu) if you are looking for an alternative. Hopefully MongoDB comes out with Realm for Flutter someday.
